docs/reference/commandline/dockerd.md

```diff
@@ -847,6 +847,17 @@ set like this:
     export DOCKER_TMPDIR=/mnt/disk2/tmp
     /usr/local/bin/dockerd -D -g /var/lib/docker -H unix:// > /var/lib/docker-machine/docker.log 2>&1
 
+Docker clients built with golang < 1.6 cannot talk to Docker daemons built
```

Comment:
This is not 100% accurate.
It's really just that older docker clients (<= 1.9) used an invalid header and go 1.6 does not allow it.